pkgs/games/atlauncher/default.nix

```
@@ -30,17 +35,17 @@ stdenv.mkDerivation (finalAttrs: {
         --suffix LD_LIBRARY_PATH : "${lib.makeLibraryPath finalAttrs.buildInputs}" ${
             lib.strings.optionalString withSteamRun ''--run "${steamrun} \\"''
           }
+
+      mkdir -p $out/share/icons/hicolor/128x128/apps
+      cp ${finalAttrs.icon} $out/share/icons/hicolor/128x128/apps/${finalAttrs.pname}.png
```

```suggestion
      cp $ICON $out/share/icons/hicolor/128x128/apps/${finalAttrs.pname}.png
```

Makes it overridable